Content.Shared/Singularity/Components/ContainmentFieldComponent.cs

```diff
@@ -18,4 +18,10 @@ public sealed partial class ContainmentFieldComponent : Component
     /// </summary>
     [DataField("maxMass")]
     public float MaxMass = 10000f;
+
+    /// <summary>
+    /// Should field vaporize garbage that collides with it?
+    /// </summary>
+    [DataField("destroyGarbage")]
```

Comment:
```suggestion
    [DataField]
```
DataField tag can be generated automatically.